ent-orm/usr/migrations/1_dummy.up.sql

```diff
@@ -0,0 +1 @@
+--Empty SQL migration to ensure the database is created by Encore
```

Comment:
What happens if we don't have this file, is there an error that happens? I feel this is a workaround for an underlying fix in Encore itself. What do you think?